extensions/pkg/controller/worker/genericactuator/actuator_delete.go

```diff
@@ -35,7 +35,11 @@ func (a *genericActuator) Delete(ctx context.Context, log logr.Logger, worker *e
 
 	workerDelegate, err := a.delegateFactory.WorkerDelegate(ctx, worker, cluster)
 	if err != nil {
-		return fmt.Errorf("could not instantiate actuator context: %w", err)
+		newError := fmt.Errorf("could not instantiate actuator context: %w", err)
```

Comment:
```suggestion
		newError := fmt.Errorf("could not instantiate actuator: %w", err)
```
When seeing `context` in error message I associate it with `context.Context` and `actuator context` reads to me as the function's fail is related to it's context variable. If `context` does not provide any benefit in the error message let's drop it